components/dashboard/src/workspaces/CreateWorkspacePage.tsx

```diff
@@ -461,6 +467,64 @@ export function CreateWorkspacePage() {
     );
 }
 
+function LocalSSHWarning(props: { className: string }) {
+    const isLocalSSHProxyEnabled = useFeatureFlag("gitpod_desktop_use_local_ssh_proxy");
+    const [isFeedbackFormVisible, setFeedbackFormVisible] = useState<boolean>(false);
+
+    if (!isLocalSSHProxyEnabled) {
+        return null;
+    }
+
+    return (
+        <div className={props.className}>
+            <Alert
+                light
+                className="bg-gray-100 dark:bg-gray-800"
+                type="info"
+                iconSize="w-8 h-8"
+                icon={<Exclamation2 className="h-full w-full" />}
+                iconColor="text-gray-500"
+            >
+                To make VS Code Desktop open seamlessly we will add a single entry to your local SSH configuration.
+                Gitpod does not read any of your existing SSH configurations.&nbsp;
+                <a
+                    className="gp-link"
+                    href="https://www.gitpod.io/docs/references/ides-and-editors/vscode#modify-ssh-config-file"
+                    target="_blank"
+                    rel="noreferrer"
+                >
+                    More Detail
+                </a>
+                <div className="flex mt-2 text-sm">
+                    Comment on&nbsp;
+                    <a
+                        href="https://github.com/gitpod-io/gitpod/issues/18109"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                        className="gp-link"
+                    >
+                        feedback issue
+                    </a>
+                    {isGitpodIo() && (
+                        <span>
+                            &nbsp;or {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+                            <a
+                                className="gp-link"
+                                // eslint-disable-next-line no-script-url
+                                href="javascript:void(0)"
+                                onClick={() => setFeedbackFormVisible(true)}
+                            >
+                                submit feedback
```

Comment:
**suggestion**: There's no need for a duplicate feedback link here as it already exists in the header. I'd suggest to remove it.